source/plugin/scheduler.rst

```diff
@@ -93,6 +93,35 @@ To cancel a task, simply call the ``Task#cancel`` method:
 
 	task.cancel();
 
+If you need to cancel the task, from within the runnable itself, you can instead opt to use a `Consumer<task>` in order to access the task from within.
+
+.. code-block:: java
+
+			Scheduler scheduler = game.getScheduler();
```

Comment:
Now we can just use `Sponge.getScheduler()`, removes the assumption that the `game` variable exists.